src/color-chart/README.md

```diff
@@ -39,6 +39,20 @@ You can also specify a whole label, and if it contains a number, the number will
 
 _(Colors courtesy of Tailwind)_
 
+### The `info` attribute
+
+You can use the `info` attribute to show information about the color scale points. Currently, the only type of information supported is color coords (in any color space), but more will be added in the future.
```

Comment:
I thought we also support deltaE?

Reply:
It's in progress. [We are iterating over it](https://github.com/color-js/elements/pull/96). Probably, it's high time to resume that iteration. 🙂